requirements.txt

```diff
@@ -0,0 +1,11 @@
+Babel==2.3.4
+decorator==4.0.10
+digExtractor>=0.1.8
+inflection==0.3.1
+jsonpath-rw==1.4.0
+jsonpath-rw-ext==1.0.0
```

Comment:
jsonpath-rw and jsonpath-rw-ext should not be in here.  can you make sure the rest of these requirements are necessary too?